contracts/Strategy.sol

```diff
@@ -300,50 +302,51 @@ contract Strategy is BaseStrategy {
         _sellBpt(_amountBpt);
     }
 
-    function claimRewards() external onlyVaultManagers {
-        _claimRewards();
-    }
-
     // claim all beets rewards from masterchef
     function _claimRewards() internal {
         if (getPendingBeets() > 0) {
-            uint256 rewardBal = balanceOfReward();
+            uint256 prevBeetsBal = balanceOfBeets();
             masterChef.harvest(masterChefPoolId, address(this));
```

Comment:
question: how is TUSD rewards claimed? Will it need to be claimed synchronously too?

Reply:
Yes.. the harvest will send both beets and tusd